Training/etc/di.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Codifi\Training\Block\Account\Dashboard\CreditHold">
```

Comment:
As you no longer use session in block, remove this proxy and use it in appropriate class